GNUmakefile

```diff
@@ -3,7 +3,7 @@ HOSTNAME=delphix.com
 NAMESPACE=dct
 NAME=delphix
 BINARY=terraform-provider-${NAME}
-VERSION=1.0.1
+VERSION=2.0.0-beta
```

Comment:
I would suggest to bump this up in a separate PR when we are ready to release. Otherwise, for local dev testing, we can overwrite 1.0.1. version right?

Reply:
Sure, will do that.

Reply:
done.